packages/core/src/visit-meta-css-dependencies.ts

```diff
@@ -0,0 +1,49 @@
+import { StylableMeta } from './stylable-meta';
+import { Imported } from './stylable-processor';
+import { StylableResolver } from './stylable-resolver';
+
+export function visitMetaCSSDependencies(
+    meta: StylableMeta,
+    onMetaDependency: (meta: StylableMeta, imported: Imported, depth: number) => void,
+    resolver: StylableResolver,
+    visited = new Set<string>(),
+    depth = 0
+): void {
+    if (visited.has(meta.source)) {
+        return;
+    }
+    visited.add(meta.source);
+    for (const imported of meta.imports) {
+        const res = resolver.resolveImported(imported, '');
+        if (res?._kind === 'css') {
+            onMetaDependency(res.meta, imported, depth);
+            visitMetaCSSDependencies(res.meta, onMetaDependency, resolver, visited, depth + 1);
+        }
+    }
+}
+
+export function visitMetaCSSDependenciesBFS(
```

Comment:
check if usage needed

Reply:
well, is it?

Reply:
We have a non BFS one that got removed so we only use this function to traverse st.css dependencies